dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/publish/servers/Telemetry10Servlet/server.xml

```diff
@@ -0,0 +1,20 @@
+<server description="Telemetry10Servlet">
+
+    <include location="../fatTestPorts.xml" />
+
+    <featureManager>
+        <feature>servlet-6.0</feature>
+        <feature>pages-3.1</feature>
+        <feature>mpTelemetry-1.0</feature>
+        <feature>componentTest-2.0</feature>
+    </featureManager>
+
+    <application id="TelemetryServletTestApp" name="TelemetryServletTestApp"
+        type="war" location="TelemetryServletTestApp.war">
+        <classloader apiTypeVisibility="+third-party" />
+    </application>
+    
+    <logging traceSpecification="TELEMETRY=all"/>
+
+    <javaPermission className="java.net.URLPermission" name="http://localhost:-/-" actions="GET:" />
```

Comment:
Please can we add a comment to say why this permission is needed?

I assume it's because the test is using HttpUrlConnection, but it'll be helpful to know why it's there if we are looking at the server.xml in future and aren't sure if it's still needed.

Reply:
Comment added